doc/doap.css

```diff
@@ -0,0 +1,88 @@
+body {
+	font-family: sans-serif
+}
+table {
+	width: 100%;
+	border-collapse: collapse;
+	clear: both;
+}
+th {
+	border: 1px solid rgb(159, 175, 190);
+	background-color: rgb(112, 128, 144);
+	color: rgb(255, 255, 255);
+	padding: 10px;
+}
+th:nth-child(1) {
+	padding-left: 2.5rem;
+	padding-right: 2.5rem;
+}
+th:nth-child(5) {
+	text-align: left;
+}
+tr {
+	border: 1px solid rgb(233, 233, 233);
+}
+tr:nth-child(odd) {
+	background-color: rgb(247, 247, 247);
+}
+tr:hover {
+	background-color: rgb(240, 240, 240);
+}
+td {
+	padding: 10px;
+	color: rgb(28, 28, 28)
+}
+td:nth-child(2) {
+    text-align: end;
+}
+td:nth-child(3) {
+    text-align: center;
+}
+td:nth-child(4) {
+    text-align: end;
+}
+a {
+	padding: 5px;
+	color: rgb(70, 130, 180);
+    text-decoration: none;
+}
+a:hover {
+	color: rgb(50, 90, 120);
+    text-decoration: none;
```

Comment:
can you use 4 spaces as indentation in the whole file?

Reply:
It is the indentation of the original file. The next step will be a mechanism to update the files coming from https://github.com/pulkomandy/xmpp-doap. Thus, I would not modify the files more than needed.

Reply:
But it seems that the indentation has changed. I will get the latest version.

Reply:
okay, so if the indentation is still mixed we should also fix that upstream

Reply:
It is already fixed upstream. I will push the change.